### PR TITLE
sending giftedChat props to renderChatFooter function prop

### PR DESCRIPTION
**Pull request by anurag-rapidue**
ExtraData property of gifted chat can't be accessed in renderChatFooter. So, Sending giftedChat Props to renderChatfooter prop function

### CODE REVIEW

File: src/GiftedChat.tsx [hunk 1]
```diff
@@ -806,7 +806,7 @@ class GiftedChat<TMessage extends IMessage = IMessage> extends React.Component<
 
   renderChatFooter() {
     if (this.props.renderChatFooter) {
-      return this.props.renderChatFooter()
+      return this.props.renderChatFooter(this.props)
```
Comment (xcarpentier) [inline src/GiftedChat.tsx]: Hi, thanks, can you please add TS definition of this props?

**Comment by xcarpentier** [comment]
any news?

File: src/GiftedChat.tsx (hunk 1 above)
Comment (anurag-rapidue) [reply src/GiftedChat.tsx]: I am new to TypeScript. However, I feel we may not need the TS definition for this props. If required, where do we need to write it.

**Comment by anurag-rapidue** [comment]
I am new to TypeScript. However, I feel we may not need the TS definition for this props. If required, where do we need to write it.

**Pull request closed without merging**